### ISSUE TITLE
Cartridge name and other text is hard to read for online map

### ISSUE THREAD

**Issue by murggel** [cgeo/WhereYouGo#352]
### Describe your problem!

I didn't play WhereYouGo in the last time, but just installed the beta-version 2022.01.22 to test it.

On my Samsung S20 the cartridge name and other text is hard to read. Changing font-size in the settings (WhereYouGo and System) didn't had any effect.
 

### How to reproduce?

![Screenshot_20220103-201955_WhereYouGo](https://user-images.githubusercontent.com/70113341/148291832-22a5899d-e711-4adf-988c-daafc73c11cf.jpg)

### Reproducible

Yes

### WhereYouGo Version

2022.01.22

### System information

```text
extracted from cgeo

Device type: SM-G980F (x1sxx, samsung)
Available processors: 8
Android version: 11
Android build: RP1A.200720.012.G980FXXUBDUI5
Screen resolution: 1080x2178px (360x726dp)
Pixel density: 3.3000002
System font scale: 1.1 / used scale: 1.1
Sailfish OS detected: false
Google Play services: enabled - 21.45.16 (150408-414021728)
```

### Additional Information

I found #76 but don't know if it is related, but appearently did not fix it on my device

**Comment by Lineflyer** [comment]
I would also assume it sabout the tile-zoom / tile size as mentioned in #76 with its reference to the c:geo issue.

We had such problem in c:geo years ago and it was fixed this way (maybe needs a newer mapsforge version).

**Comment by Lineflyer** [comment]
If someone wants to work on this:
The solution is described in https://github.com/cgeo/cgeo/issues/7750 and the linked commits / PRs.

**Comment by Lineflyer** [comment]
An extract of one issue comment there by @rsudev and base for the later fix by @jonas-koeritz 

> After researching for a while, I think I now know the background for this issue.
The zoom level relates to the tile the map is drawn with.
> - Google map always uses tile size 256
> - Mapsforge online always uses tile size 256
> - Mapsforge offline uses a tile size depending on the the resolution (I tested mdpi through to xxhdpi):
>    - Minimum (up to hdpi) is 256
>    - xhdpi 512
>    - xxhdpi 768

**Comment by Lineflyer** [comment]
@murggel 
Yes, #76 is the same IMHO. I copied the relevant information to here, renamed your issue and closed #76.

**Comment by andrixnet** [comment]
I was about to say that 2022.01.14 version still has this. Adding myself to this thread.
